apps/web/src/app/api/v0/ping/route.ts

```diff
@@ -0,0 +1,34 @@
+import { env } from "@/env.mjs";
+import { Tinybird, publishPingResponse } from "@openstatus/tinybird";
+import { Redis } from "@openstatus/upstash";
+
+const tb = new Tinybird({ token: env.TINY_BIRD_API_KEY });
+const redis = Redis.fromEnv();
+
+// include ratelimit
+
+async function monitor(statusCode: number) {
+  return await publishPingResponse(tb)({
+    id: "openstatus",
+    timestamp: Date.now(),
+    statusCode,
+  });
+}
+
+export async function GET(req: Request) {
+  const RANDOM = Math.random() > 0.5;
+  try {
+    // REMINDER: connect tinybird, upstash and planetscale
+    await redis.ping();
+    const res = new Response("OK", { status: 200 });
+    await monitor(200);
+    if (RANDOM) {
```

Comment:
I dont get it ?

Reply:
This is just to a random error for testing purposes. I updated the equation to have a .1% chance to fail.